dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -132,6 +132,19 @@ def _merge_words(self, other, merged_profile):
                     merged_profile.word_count[word] += additive_words[word]
                 else:
                     merged_profile.word_count[word_lower] += additive_words[word]
+
+    @staticmethod
+    def _merge_vocab(vocab_count1, vocab_count2):
+        """
+        Merges the vocab counts of two TextProfiler profiles
+
+        :param vocab_count1: vocab count of the first profile
+        :param vocab_count2: vocab count of the second profile
+        :type vocab_count1: Counter()
+        :type vocab_count2: Counter()
+        :return:
```

Comment:
Should say what it returns here. 
Otherwise, PR looks good, will approve as long as speed is all good.